ui/src/views/Dashboard.vue

```diff
@@ -1231,9 +1231,24 @@ export default {
       this.migrationUpdate(this.currentApp, "start");
       this.hideStartMigrationModal();
     },
+    cleanError(){
```

Comment:
This function doesn't clear all errors, I would rename it to `clearValidationErrors`